src/erlcloud_kms.erl

```diff
@@ -986,10 +986,19 @@ kms_result_fun(#aws_request{response_type = error, error_type = aws} = Request)
     Request#aws_request{should_retry = false}.
 
 
-%
-% Private
-%
-request(Config, Operation, Body) ->
+%%%------------------------------------------------------------------------------
+%%% Internal Functions
+%%%------------------------------------------------------------------------------
+
+kms_request(Config, Operation, Body) ->
+    case erlcloud_aws:update_config(Config) of
```

Comment:
It looks like the right thing to do. It just ensures that the process dictionary is up to date with the latest config, if a custom config were passed to one of the calls.